babel.config.js

```diff
@@ -29,6 +30,18 @@ module.exports = api => {
                     ],
                 ],
             },
+            {
+                test: ['./src/third-party/doc/2.84.0/**'],
```

Comment:
We probably don't want to tie it to a specific version, can we have the regex match the contents of `./src/third-party/doc/**`?